numpyro/infer/autoguide.py

```diff
@@ -1250,13 +1250,16 @@ def local_model(theta):
         during partial momentum refreshments in HMC. Defaults to 0.9.
     :param float init_scale: Initial scale for the standard deviation of the variational
         distribution for each (unconstrained transformed) local latent variable. Defaults to 0.1.
+    :param str subsample_plate: Optional name of the subsample plate site. This is required
+        when the model does not have subsample plate (like in VAE settings).
```

Comment:
does not?

Reply:
just revised it to clarify that this is required when the model has a subsample plate without `subsample_size` specified.